### PR TITLE
add a way to expose the underlying gRPC Server and to decorate grpc c…

### PR DESCRIPTION
**Pull request by michalszynkiewicz**
Adding a bit more flexibility to enable implementing Quarkus live reload.
If this one is okay, I will prepare a PR for master too.
I did it for 3.9 because this is what we currently use for quarkus-grpc

### CODE REVIEW

**Comment by vietj** [comment]
@michalszynkiewicz can you add some integration test to ensure the behaviour ?

**Comment by michalszynkiewicz** [comment]
@vietj I'm not sure if that's the simplest way to test it but I added some test

**Comment by michalszynkiewicz** [comment]
I don't understand what happened on Travis. It didn't run the newly added test at all and failed on a test that seems irrelevant to my changes. Cannot reproduce the failure locally. And don't see a button to retrigger the check :(

**Comment by vietj** [comment]
it seems ok now

**Comment by michalszynkiewicz** [comment]
I did a small formatting change to retrigger the build and it went well this time.
It failed here: https://travis-ci.org/github/vert-x3/vertx-grpc/builds/680434848

**Comment by vietj** [comment]
it looks like either the test is wrong or there is a bug

File: src/main/java/io/vertx/grpc/VertxServerBuilder.java [hunk 1]
```diff
@@ -143,8 +145,13 @@ public VertxServerBuilder useSsl(Handler<HttpServerOptions> handler) {
     return this;
   }
 
+  public VertxServerBuilder commandDecorator(Consumer<Runnable> commandDecorator) {
```
Comment (vietj) [inline src/main/java/io/vertx/grpc/VertxServerBuilder.java:148]: Doc is missing

Comment (michalszynkiewicz) [reply src/main/java/io/vertx/grpc/VertxServerBuilder.java:155]: done

**Comment by michalszynkiewicz** [comment]
@vietj if you find the change okay, could we have a release with it? It would enable us to have live reload for Quarkus gRPC

**Comment by vietj** [comment]
@michalszynkiewicz this will be released in 3.9.1 I think

**Comment by vietj** [comment]
it is merged @michalszynkiewicz you can port to master

**Comment by michalszynkiewicz** [comment]
thanks @vietj! Ported to master: https://github.com/vert-x3/vertx-grpc/pull/66

**Pull request merged**